### PR TITLE
[batch] Remove all v2 billing tables

### LINKED ISSUE #13656: [batch] eliminate _v2 billing tables
### What happened?

Let's pay our technical debt and delete a bunch of code!

### Version

0.2.122

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jigold**
Fixes #13656.  This PR removes all triggers and tables associated with the v2 tables. Stacked on #13892.

### CODE REVIEW

**Review by daniel-goldstein — APPROVED** [review]

**Comment by danking** [comment]
Any reason to not merge this today?

**Comment by jigold** [comment]
If you'll be around this afternoon / evening, then that's fine. We should do this before we merge job groups to save storage space.

**Comment by jigold** [comment]
I have PT at 3:40 and am busy in the evening.

**Pull request merged**
